app/Views/CoaBooking/_Layout.cshtml

```diff
@@ -27,10 +27,12 @@
 
 @section Scripts {
     <script src="/scjob/js/coa.js" asp-append-version="true"></script>
+    <script src="/scjob/dist/vue/coa.js" asp-append-version="true"></script>
```

Comment:
Can we rename the folder in /app/ClientSrc/vue to make them more distinguishable. It should be possible to have multiple vue apps for both sc and coa, so naming the folder "coa" makes it look like there can only be one coa app.  

- coa-application-type-picker  (the new one)
- sc-hearing-time-picker  (the existing one)

Reply:
We can have just one `.js` file that includes several `.vue` files, so I figured we could organize them into COA and SC folders... but there could also be components used in both places. 🤔 Maybe we'll get some inspiration on how to organize components when we have more than just one on each site, though. 

I'll give each component its own folder and index.js file like you suggested 👍 Seems the clearest approach for now

Reply:
I renamed them to HearingTimeSelect and ApplicationTypeSelect and updated the related files as needed